addons/misc/CfgWeapons.hpp

```diff
@@ -16,4 +16,43 @@ class CfgWeapons
             scope = 0;
         };
     };
+
+    class ItemCore;
+    class kat_Armband_Red_Cross_Item: ItemCore {
+
+        detectRange = -1;
+        author = "Miss Heda, vccv9040 (Swedish Forces Pack)";
+        simulation = "ItemMineDetector";
+        displayName = CSTRING(Armband_Red_Cross_ItemName);
+        scope = 2;
+        cost = 2;
```

Comment:
cost = 200000;	// How likely the enemies attack this character among some others.
wiki says its used only for character config https://community.bistudio.com/wiki/Arma_3:_Characters_And_Gear_Encoding_Guide

Reply:
The cost is for curator. How much it costs to spawn this item